### DISCUSSION TITLE
Change size of the icons in calendar mode

### DISCUSSION THREAD

**Discussion by gabstrusi** [totaldebug/atomic-calendar-revive#1250] [Showroom]
Hi,

Could you please help me with the code necessary to change the size of the icons using card-mod?

<img width="560" alt="Screenshot 2023-10-23 at 15 03 44" src="https://github.com/totaldebug/atomic-calendar-revive/assets/147725502/829be532-5c3f-41bb-91e9-dac1c3462a60">

Thank you

**Comment by marksie1988** [comment]
check out this video on how to use card_mod and find out where to get these from: https://www.youtube.com/watch?v=-5MKd7LY-oc

```
card_mod:
  style: | 
    .calIcon {
      --mdc-icon-size: 15px !important;
    }
```

**Reply by gabstrusi** [reply]
Great, thank you so much!
